screens/SendSuccessScreen.js

```diff
@@ -42,6 +42,7 @@ const IconImage = styled(Image)`
   height: 64;
   border-radius: 32;
   overflow: hidden;
+  border: 2px solid ${props => props.theme.bg900};
```

Comment:
Invalid image property.  Looks better, but linter complains.  
Might need to move this to a parent div, or some other solution

Reply:
removing for now.